src/java/org/apache/cassandra/repair/consistent/LocalSessions.java

```diff
@@ -1021,6 +1021,7 @@ public void handleStatusResponse(InetAddressAndPort from, StatusResponse respons
         }
         else
         {
+            session.setLastUpdate();
             logger.debug("Received StatusResponse for repair session {} with state {}, which is not actionable. Doing nothing.", sessionID, response.state);
```

Comment:
We are technically not `Doing nothing` anymore right?

Reply:
I think icwym. You're probably referring to the junit being named 'noop'? It depends a lot on how you parse that in your brain imo. I read it as it's noop in terms of the status of the repair, which doesn't change. You probably read it as a pure 100% noop whereas now we're updating a timestamp? Some other people read junits as a pseudo-doc of the expected behavior of what a noop status operation should do :shrug: I think we're good and we avoid adding an extra almost identical test.